Project/Project.md

```diff
@@ -0,0 +1,116 @@
+data with simple arrows, and the gentrification level data can be
+presented in the different scales of color in the tract (can do these
+steps in geom\_sf (aes(fill=…)) and geom\_segment())
+
+## Example 3
+
+![crime
+map](https://bookdown.org/fis/social-life-of-neighborhoods/images/crime_heat_map.gif)
+Alternatively, this kind of crime map can also be used in my project.
+Crime incidents can be represented as points in the map, and by creating
+the heat map, the incidents can be identified more easily. Multiple
+incidents that happen in the same tract can be specified in the pop-up
+boxes by using addCircleMarkers(popup= \~paste0(crime type,
+incident\_date, …) in the leaflet package.
+
+# Proposed data sources
+
+Buffalo crime records at the census tract level from 2011-2019 will be
+geocoded according to the longitude and latitude in the Open Data
+Buffalo database. The Buffalo census tract data from 2011 to 2019 used
+for measuring gentrification will be retrieved from the “tidycensus”
+package in R.
+
+# Proposed methods
+
+1.  I am going to use the tidycensus package to get the data for
+    measuring the degree of gentrification from 2011 to 2019 acs. I will
+    select and calculate the variables that I need to measure
+    gentrification according to the gentrification index (Appendix A).
+    Writing a loop to create a data list for each year at both tract and
+    county levels. I manually add the education variables in the 2011
+    and 2010 data from the social explorer website since the tidycensus
+    doesn’t have the variable that I need in these two years.
+2.  Creating a gentrification\_degree df: I am going to assign a
+    gentrification degree to each census tract (No
+    gentrification-related changes, susceptible to gentrification,
+    middle stage, late stage, ongoing gentrification, gentrified) (in
+    Appendix B).
+3.  Merge with the crime data (gentri\_crime df): download the crime
+    data from the data.buffalony.gov in the csv. format, and use the
+    merge() function to merge the crime data with the spatial census
+    data.
+4.  Creating the gentrification degree map (code maybe like this):
+    Gentrification\_df %&gt;% group\_by(Year) ggplot(gentri\_crime,
+    aes(fill=gentrification\_degree))+ geom\_sf()+
+    scale\_fill\_viridis\_c()
+5.  Adding crime data on the map: Using leaflet(gentri\_crime) %&gt;%
+    addProviderTiles(“CartoDB”(maybe?)) %&gt;%
+    addCircleMarkers(label=\~paste0(tract\_names, …), radius=…,
+    color=\~pal(…)) to point out the crime data.
+    crime\[,c(“x”,“y”)\]=st\_coordinates(st\_centroid(crime))?
+6.  Use GGally package to build correlograms, and the ggplot2 package to
+    color and split the year group. Corr: ggcorr(gentri\_crime, method =
+    c(“everything”, “pearson”)) \#Can I make dynamic maps among
+    different years?
+
+# Expected results
+
+In the map that I am going to make, I expect the crime rate will be
```

Comment:
as we talked about before, be careful with concluding cause and effect in any correlative relationship you may find. For example, as a neighborhood gentrifies, police may patrol it less, leading to less _recorded_ crime.  I'm not an expert in such things - but I know it's a tricky thing to study.